src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -3075,6 +3080,9 @@ func LaunchCustomQemu(ctx context.Context, path string, params []string, fds []*
 	}
 
 	go func() {
+		if logFunc != nil {
+			logFunc(cmd.Process.Pid, stderrIn)
+		}
```

Comment:
OK, that part of the commit answers my comment on what you intended to do with this goroutine.